README.rst

```diff
@@ -106,20 +106,37 @@ Following a conflicted “rebase” operation in your source control tool, run i
 
     $ python manage.py rebase-migration <app_label>
 
+Note rebasing the migration might not always be the *correct* thing to do.
+If the migrations in main and feature branches have both affected the same models, rebasing the migration on the end may not make sense.
+However, such parallel changes would *normally* cause conflicts in your models files or other parts of the source code as well.
+
 Let's walk through an example using Git, although it should extend to other source control tools.
 
-Imagine you were working on your project's ``books`` app in a feature branch and created a migration called ``0002_longer_titles``.
+Imagine you were working on your project's ``books`` app in a feature branch called ``titles`` and created a migration called ``0002_longer_titles``.
 Meanwhile a commit has been merged to your ``main`` branch with a *different* 2nd migration for ``books`` called ``0002_author_nicknames``.
 Thanks to django-linear-migrations, the ``max_migration.txt`` file will show as conflicted between your feature and main branches.
 
-You can start to fix the conflict by pulling your latest ``main`` branch, then rebasing your ``titles`` branch on top of it.
-When you do this, Git will report the conflict:
+You start the fix by reversing your new migration from your local database.
+This is necessary since it will be renamed after rebasing and seen as unapplied.
+You do this by switching to the feature branch ``titles`` migrating back to the last common migration:
+
+.. code-block:: console
+
+    $ git switch titles
```

Comment:
it's surprising to me that they might not be on the titles branch here

perhaps this comes up because you didn't start the example with a git merge/rebase

Reply:
Just wanted to be explicit, haven't encountered anything in particular